server/models/Profile.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+const reqString = {
+    type: String,
+    require: true
+};
+
+const ProfileSchema = new mongoose.Schema({
+    userId: {
+        type: mongoose.Schema.Types.ObjectId,
+        require: true,
+        ref: 'user'
+    },
+    firstName: reqString,
+    lastName: reqString,
+    email: reqString,
+    description: {
+        type: String,
+        maxlength: 300
+    },
+    gender: String,
```

Comment:
Since `gender` is a dropdown in the mocks, you can use `enum` to specify the allowed values for gender